backend/tests/test_logout.py

```diff
@@ -18,3 +31,33 @@ def test_logout_user_without_token(client):
     response = rv.get_json()
     assert rv.status_code == HTTPStatus.UNAUTHORIZED
     assert response["msg"] == "Missing Authorization Header"
+
+
+def test_logout_twice(app, client, access_token):
+    """Test logout twice."""
+    with app.app_context():
+        rv = client.delete(
+            "/auth/logout/", headers={"Authorization": "Bearer {}".format(access_token)}
+        )
+        assert rv.status_code == HTTPStatus.OK
+        rv = client.delete(
+            "/auth/logout/", headers={"Authorization": "Bearer {}".format(access_token)}
+        )
+        tokens = JWTToken.query.all()
+    response = rv.get_json()
```

Comment:
Nitpick: Could you reaname variables on more meaningful? E.g. `rv` on `response` and `response` ( in `response = rv.get_json()`) on `payload`